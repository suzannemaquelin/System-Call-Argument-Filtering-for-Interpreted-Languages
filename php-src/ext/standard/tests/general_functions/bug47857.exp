bool(true)

Deprecated: Non-static method foo::bar() should not be called statically in %sbug47857.php on line %d
ok
bool(false)

Fatal error: Uncaught Error: Non-static method Exception::getMessage() cannot be called statically in %sbug47857.php:%d
Stack trace:
#0 {main}
  thrown in %sbug47857.php on line %d