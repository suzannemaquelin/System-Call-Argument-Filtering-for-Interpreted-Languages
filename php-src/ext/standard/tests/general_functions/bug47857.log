
---- EXPECTED OUTPUT
bool(true)

Deprecated: Non-static method foo::bar() should not be called statically in %sbug47857.php on line %d
ok
bool(false)

Fatal error: Uncaught Error: Non-static method Exception::getMessage() cannot be called statically in %sbug47857.php:%d
Stack trace:
#0 {main}
  thrown in %sbug47857.php on line %d
---- ACTUAL OUTPUT
Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
bool(true)

Deprecated: Non-static method foo::bar() should not be called statically in /root/php-src/ext/standard/tests/general_functions/bug47857.php on line 8
ok
bool(false)

Fatal error: Uncaught Error: Non-static method Exception::getMessage() cannot be called statically in /root/php-src/ext/standard/tests/general_functions/bug47857.php:10
Stack trace:
#0 {main}
  thrown in /root/php-src/ext/standard/tests/general_functions/bug47857.php on line 10
---- FAILED
