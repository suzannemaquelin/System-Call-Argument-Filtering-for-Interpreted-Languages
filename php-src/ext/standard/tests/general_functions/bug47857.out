Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
bool(true)

Deprecated: Non-static method foo::bar() should not be called statically in /root/php-src/ext/standard/tests/general_functions/bug47857.php on line 8
ok
bool(false)

Fatal error: Uncaught Error: Non-static method Exception::getMessage() cannot be called statically in /root/php-src/ext/standard/tests/general_functions/bug47857.php:10
Stack trace:
#0 {main}
  thrown in /root/php-src/ext/standard/tests/general_functions/bug47857.php on line 10